delete_student.py

```diff
@@ -0,0 +1,25 @@
+def random_st_delete(list_of_student):
+    for i in range(len(list_of_student)):
+        print(i+1, list_of_student[i])
+    print('are all students here? Answer y/n')
+    x = input()
+    if x == 'n':
+        print('who is absent today? print their №')
+        absent_student = [int(i) for i in input().split()]
+        for j in range(len(absent_student)):          
```

Comment:
`j` will be 0, 1, 2 for `absent_student = [3, 4, 5]` is this the intendent behavior?